client/my-sites/posts/post-share.jsx

```diff
@@ -212,6 +226,7 @@ export default connect(
 		const userId = getCurrentUserId( state );
 
 		return {
+			planHasRepublicizeFeature: hasFeature( state, siteId, FEATURE_REPUBLICIZE ),
```

Comment:
So some of the main concerns here for me would be:
1. Some features are legacy add-ons (space, video etc) so other folks in Calypso using these checks would be incorrect for anyone that has the legacy add-ons, but not a plan.
2. Calypso is not the only client of the rest-api, If we add new plan features that aren't represented in the api as the source of truth, I think our mobile implementations will have a hard time catching up
3. Using this as a check to display nudges is probably OK, as false negatives aren't as bad if I see an ad for a thing I already have, but we're also using this as a feature gate here which I think is incorrect.